packages/runtime/plugin-runtime/src/router/runtime/plugin.node.tsx

```diff
@@ -144,47 +135,40 @@ export const routerPlugin = ({
           context.routerContext = routerContext;
           context.routes = routes;
           // set routeManifest in context to be consistent with csr context
-          context.routeManifest = context.ssrContext!
-            .routeManifest as RouteManifest;
+          // context.routeManifest = context.ssrContext!
```

Comment:
It seems that there are some comments left here

Reply:
Got it